doc/changes/changes_2.1.2.md

```diff
@@ -0,0 +1,21 @@
+# Exasol Test Setup Abstraction Java 2.1.2, released 2024-??-??
```

Comment:
```suggestion
# Exasol Test Setup Abstraction Java 2.1.2, released 2024-??-??
```
Since I am guessing you want to use that as basis for something else, shouldn't you make a release?

Reply:
No release planned, just PK update.